### PR TITLE
kubectl v1.20.0 pre-packaged binaries x86_64-* only

### PR DESCRIPTION
**Pull request by kolia**
Gave up on trying to compile kubernetes, this just downloads pre-compiled binaries.

### CODE REVIEW

File: K/kubectl/build_tarball.jl [hunk 1]
```diff
@@ -0,0 +1,52 @@
+using BinaryBuilder
+
+# Collection of sources required to build HDF5
+name = "kubectl"
+version = v"1.20.0"
+
+sources = [
+    FileSource("https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/linux/amd64/kubectl", "a5895007f331f08d2e082eb12458764949559f30bcc5beae26c38f3e2724262c"; filename="kubectl.linux"),
+    FileSource("https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/darwin/amd64/kubectl", "82046a4abb056005edec097a42cc3bb55d1edd562d6f6f38c07318603fcd9fca"; filename="kubectl.darwin"),
+    FileSource("https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/windows/amd64/kubectl.exe", "ee7be8e93349fb0fd1db7f5cdb5985f5698cef69b7b7be012fc0e6bed06b254d"; filename="kubectl.exe"),
```
Comment (giordano) [inline K/kubectl/build_tarball.jl:10]:
```suggestion
    FileSource("https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/linux/amd64/kubectl", "a5895007f331f08d2e082eb12458764949559f30bcc5beae26c38f3e2724262c"; filename="x86_64-linux-gnu-kubectl"),
    FileSource("https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/darwin/amd64/kubectl", "82046a4abb056005edec097a42cc3bb55d1edd562d6f6f38c07318603fcd9fca"; filename="x86_64-apple-darwin-kubectl"),
    FileSource("https://storage.googleapis.com/kubernetes-release/release/v1.20.0/bin/windows/amd64/kubectl.exe", "ee7be8e93349fb0fd1db7f5cdb5985f5698cef69b7b7be012fc0e6bed06b254d"; filename="x86_64-w64-mingw32-kubectl"),
```

File: K/kubectl/build_tarball.jl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    chmod +x ${bindir}/kubectl.exe
+
+elif [[ ${target} == x86_64-apple* ]]; then
+    mv kubectl.darwin ${bindir}/kubectl
+    chmod +x ${bindir}/kubectl
+
+else
+    mv kubectl.linux ${bindir}/kubectl
+    chmod +x ${bindir}/kubectl
+fi
```
Comment (giordano) [inline K/kubectl/build_tarball.jl:32]:
```suggestion
mv "${target}-kubectl" "${bindir}/kubectl${exeext}"
chmod +x "${bindir}/kubectl${exeext}"
```

File: K/kubectl/build_tarball.jl [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64),
+    MacOS(:x86_64),
+    Windows(:x86_64),
```
Comment (giordano) [inline K/kubectl/build_tarball.jl:41]:
```suggestion
    Platform("x86_64", "linux"; libc="glibc"),
    Platform("x86_64", "macos"),
    Platform("x86_64", "windows"),
```

File: K/kubectl/build_tarball.jl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+]
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("kubectl", :kubectl),
+]
+
+dependencies = []
```
Comment (giordano) [inline K/kubectl/build_tarball.jl:49]:
```suggestion
dependencies = Dependency[]
```

**Review by giordano — COMMENTED** [review]
This is much simpler

File: K/kubectl/build_tarball.jl [hunk 5]
```diff
@@ -0,0 +1,42 @@
+using BinaryBuilder
+
+# Collection of sources required to build HDF5
```
Comment (ericphanson) [inline K/kubectl/build_tarball.jl:3]:
```suggestion

```

**Comment by giordano** [comment]
https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=8163&view=logs&jobId=95743c6e-4af7-5d83-3f07-511094317aef&j=96c13d9d-dfd2-51e5-2f76-4a0b3ce09abc&t=09532434-8b0a-5f0e-118d-a3181cde5ec8
```
Considering K/kubectl
 --> K/kubectl does not contain a build_tarballs.jl, moving up a directory
 --> Skipping as we could not find a build_tarballs.jl
```

**Pull request merged**